x/concentrated-liquidity/lp.go

```diff
@@ -336,23 +337,27 @@ func (k Keeper) initializeInitialPositionForPool(ctx sdk.Context, pool types.Con
 	// Calculate the spot price and sqrt price from the amount provided
 	initialSpotPrice := amount1Desired.ToDec().Quo(amount0Desired.ToDec())
 
-	// Calculate the initial tick from the initial spot price
-	// We use banker's rounding here so that the tick is rounded to
-	// the nearest value relative to the true value given the tick spacing of 1.
-	initialTick, err := math.PriceToTickRoundBankers(initialSpotPrice, pool.GetTickSpacing())
+	initialCurSqrtPrice, err := initialSpotPrice.ApproxSqrt()
 	if err != nil {
 		return err
 	}
 
-	// Since tick can be rounded due to tick spacing
-	// we calculate the initial sqrt price from the initial tick
-	initialSqrtPrice, err := math.TickToSqrtPrice(initialTick)
+	// Calculate the initial tick from the initial spot price
+	// We round dow here so that the tick is rounded to
```

Comment:
I swear I tried so hard to avoid making a typo in this PR, you see everything 👀 

I need a Grammarly extension that only lints comments:
![image](https://user-images.githubusercontent.com/34196718/235269646-5b28506c-8376-4a1c-908e-a5e005b24fe3.png)